torch/_dynamo/variables/tensor.py

```diff
@@ -184,11 +184,16 @@ def var_getattr(self, tx, name):
         if result is not None and self.source is not None:
             result = result.add_guard(self.make_guard(GuardBuilder.TYPE_MATCH))
 
-        # It's hard to get resize_() on graph input work properly across
+        # It's hard to get inplace view (metadata mutation) on graph input work properly across
         # dynamo/aot/inductor, just fall back.
-        if name in ("resize_", "unsqueeze_", "resize_as_") and self.source is not None:
-            # Delay the graph break to the actual call of unsqueeze_/resize_/resize_as_
-            return variables.misc.DelayGraphBreakVariable()
+        if self.source is not None:
+            try:
+                fn = getattr(torch.ops.aten, name)
+                if torch.Tag.inplace_view in getattr(fn, fn.overloads()[0]).tags:
+                    # Delay the graph break to the actual call of unsqueeze_/resize_/resize_as_ etc.
+                    return variables.misc.DelayGraphBreakVariable()
+            except AttributeError:
+                pass
```

Comment:
can you do "if" check instead of try-catch?

Reply:
Done